sdk/cosmos/azure-cosmos/azure/cosmos/database.py

```diff
@@ -584,8 +688,47 @@ def replace_container(  # pylint:disable=docstring-missing-param
         analytical_storage_ttl: Optional[int] = None,
         computed_properties: Optional[List[Dict[str, str]]] = None,
         full_text_policy: Optional[Dict[str, Any]] = None,
+        return_headers: Optional[bool] = False,
         **kwargs: Any
     ) -> ContainerProxy:
+        ...
+
+    @overload
+    def replace_container(  # pylint:disable=docstring-missing-param
+            self,
+            container: Union[str, ContainerProxy, Mapping[str, Any]],
+            partition_key: PartitionKey,
+            indexing_policy: Optional[Dict[str, Any]] = None,
+            default_ttl: Optional[int] = None,
+            conflict_resolution_policy: Optional[Dict[str, Any]] = None,
+            populate_query_metrics: Optional[bool] = None,
+            *,
+            initial_headers: Optional[Dict[str, str]] = None,
+            analytical_storage_ttl: Optional[int] = None,
+            computed_properties: Optional[List[Dict[str, str]]] = None,
+            full_text_policy: Optional[Dict[str, Any]] = None,
+            return_headers: Optional[bool] = True,
+            **kwargs: Any
+    ) -> CosmosDict:
+        ...
+
+    @distributed_trace
+    def replace_container(  # pylint:disable=docstring-missing-param
+        self,
+        container: Union[str, ContainerProxy, Mapping[str, Any]],
+        partition_key: PartitionKey,
+        indexing_policy: Optional[Dict[str, Any]] = None,
+        default_ttl: Optional[int] = None,
+        conflict_resolution_policy: Optional[Dict[str, Any]] = None,
+        populate_query_metrics: Optional[bool] = None,
```

Comment:
If we are adding overloads - we can actually use this opportunity to clean up some of these deprecated positional parameters!